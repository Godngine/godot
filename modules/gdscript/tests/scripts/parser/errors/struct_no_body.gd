struct Test:


func test():
    pass
