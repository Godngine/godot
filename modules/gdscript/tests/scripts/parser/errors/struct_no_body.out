GDTEST_PARSER_ERROR
Expected indented block after struct declaration.
